Source/Morris.Reducible.Tests/ReducerTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Xunit;
+
+namespace Morris.Reducible.Tests
+{
+	public class ReducerTests
+    {
+	    private record CounterState(int Counter);
+	    private record IncrementCounter(int Amount);
+
+
+		[Fact]
+	    public void WhenSimpleReducerInvoked_ThenStateIsUpdated()
+	    {
+			//Given
+			var initialState = new CounterState(0);
+			var action = new IncrementCounter(1);
+			var counterIncrementCounterReducer = Reducer
+			 .Given<CounterState, IncrementCounter>()
+			 .Then((state, delta) => (true, state with { Counter = state.Counter + delta.Amount}));
+
+			//When
+			(bool changed, var counterState) = counterIncrementCounterReducer.Invoke(initialState, action);
```

Comment:
You don't need to call `.Invoke` on these. Just `xxxReducer(state, delta)`

Reply:
Fixed!